tensai-api/src/main/java/dev/phomc/tensai/keybinding/SimpleKeyBindingManager.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of tensai, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) 2023 PhoMC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package dev.phomc.tensai.keybinding;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SimpleKeyBindingManager implements KeyBindingManager {
+	private final Map<String, KeyBinding> keyBindings = new HashMap<>();
+
+	@Override
+	public void registerKeyBinding(KeyBinding keyBinding) {
```

Comment:
Don't forget to broadcast new key binding to all online players when a mod/plugin register a new one mid-game.

Reply:
I don't allow that

Reply:
Some users tends to use /reload so registering keys after reload is needed. Plus you also need to unregister keys when plugin is being disabled to ensure we won't get duplicated keys.